client/trino-client/src/main/java/io/trino/client/ClientStandardTypes.java

```diff
@@ -40,11 +43,14 @@ public final class ClientStandardTypes
     public static final String ARRAY = "array";
     public static final String MAP = "map";
     public static final String JSON = "json";
+    public static final String JSON_2016 = "json2016";
```

Comment:
This is an internal type, not meant to be exposed to clients

Reply:
It's not exposed. It's here so the files are the same (server/client ones) for easier comparison